src/main/java/at/hannibal2/skyhanni/config/features/inventory/InventoryConfig.java

```diff
@@ -188,6 +189,15 @@ public String toString() {
     @FeatureToggle
     public boolean shiftClickForEquipment = false;
 
+    @Expose
+    @ConfigOption(name = "Dojo Rank Display", desc = "Display your rank, score and actual belt in the Challenges inventory.")
```

Comment:
Since this is in the inventory category, maybe include the word crimson isle in here. for users that dont know what/where dojo is and for search optimisations. (or move the setting to crimson isle config idk)